interactive_ai/workflows/train/job/commands/create_task_train_dataset_command.py

```diff
@@ -42,6 +45,7 @@ def __init__(
         self.dataset: Dataset = NullDataset()
         self.max_training_dataset_size = max_training_dataset_size
         self.reshuffle_subsets = reshuffle_subsets
+        self.training_configuration = training_configuration
```

Comment:
Does `training_configuration` also include the `reshuffle_subsets` option? If so, which one takes precedence?